modules/environment/util/docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM amazonlinux:2
+RUN yum update -y
+RUN yum install -y \
+        shadow-utils.x86_64 \
+        zip unzip \
+        gcc-c++ \
+        make \
+        openssl-devel \
+        zlib-devel \
+        readline-devel \
+        git \
+        blas blas-devel \
+        lapack lapack-devel \
+        wget \
+        zsh \
+        which \
+        tmux
```

Comment:
Include `tar` package required for vscode remote dev

Reply:
> Add in a python package to support vscode remote development on container.

Which package are you referring to? tar or something else?